units/user@.service.in

```diff
@@ -23,6 +23,7 @@ ExecReload=systemctl --user daemon-reload
 Slice=user-%i.slice
 KillMode=mixed
 Delegate=pids memory cpu
+DelegateControlGroupSuffix=init.scope
```

Comment:
(btw, we should do something similar in systemd-nspawn@.service, but it's a bit more involved there, since we first need to teach nspawn to recognize if it is invoked like that. maybe leave for another PR)

Reply:
Note taken. (Not added to current patchset.)